### ISSUE TITLE
Eliminate the "zip" form factor of the TCK 

### ISSUE THREAD

**Issue by scottkurz** [WASdev/standards.jsr352.tck#1]
The zip doesn't seem to be adding much.  Consumers could just use a well-documented release version from Maven central.     Seems the biggest TODO here is simply to document the needed series of steps (probably would need to be against a 1.1-b01 TCK release - will leave the 1.0 TCK alone).

**Comment by scottkurz** [comment]
This really depends on issue #11

**Comment by scottkurz** [comment]
Superseded by Jakarta Batch
